addons/hr_holidays/views/hr_leave_accrual_views.xml

```diff
@@ -176,6 +176,7 @@
                                 <field name="accrual_validity_count"/>
                                 <field name="accrual_validity_type"/>
                                 <field name="cap_accrued_time"/>
+                                <field name="accrual_validity" invisible="1"/>
```

Comment:
seems useless

Reply:
without it, invisible condition isn't working